mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -190,10 +192,24 @@ def start_conn_process():
                             result = value
                             break
                         elif key == '__reset_dut':
-                            # Disconnecting and re-connecting comm process will reset DUT
+                            # Disconnect to avoid connection lost event
                             dut_event_queue.put(('__host_test_finished', True, time()))
```

Comment:
That will reload the host test. Remember host test manages the state for the target. Hence it should not be reloaded.